Rainbowth.sublime-settings

```diff
@@ -2,6 +2,7 @@
   "palettes":
   {
     "default": ["red", "orange", "yellow", "green", "blue", "indigo", "violet"],
-    "Tomorrow Night": ["#c66", "#de935f", "#ee6", "#b5bd68", "#81a2be", "#b294bb", "#ff69b4", "#8a2be2"]
+    "Tomorrow Night": ["#c66", "#de935f", "#ee6", "#b5bd68", "#81a2be", "#b294bb", "#ff69b4", "#8a2be2"],
+    "Monokai": ["#AE81FF", "#66D9EF", "#A6E22E", "#FD971F", "#F92672"]
```

Comment:
This is for users who use the popular Soda look for sublime with the "Monokai" color scheme. 

Looks like this:
![Example](http://i.imgur.com/YBEHWfR.png)